apifest-oauth20/src/main/java/com/apifest/oauth20/RedisDBManager.java

```diff
@@ -42,10 +44,13 @@
     static {
         sentinels = new HashSet<String>();
         String[] sentinelsList = OAuthServer.getRedisSentinels().split(",");
+        JedisPoolConfig poolConfig = new JedisPoolConfig();
+        poolConfig.setMaxActive(30);
```

Comment:
May be that "30" should be configurable? Or at least not a magic number here - use a static variable instead.